draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -416,7 +416,10 @@
                 offerer MUST insert an SDP 'dtls-id' attribute with the previously assigned value in the offer.
                 In addition, the offerer MUST insert an SDP 'setup' attribute with a value that does
                 not change the previously negotiated DTLS roles, and one or more SDP 'fingerprint' attributes with
-                values that do not change the previously sent fingerprints, in the offer.
+                values that do not change the previously sent fingerprints, in the offer. The value of the 'setup'
+                attribute SHOULD be set to 'actpass', in order to allow the answerer to establish a new DTLS
+                association with a different role, but MAY be set to the current negotiated role ('active' or
+                'passive').

```

Comment:
What we are trying to say:
1. MUST insert 'setup' attribute
2. SHOULD insert setup attribute with value 'actpass', in order to allow the answerer to establish a new DTLS association with a different setup role
4. MAY insert setup attribute with currently negotiated value ('active' or 'passive'), but MUST NOT insert a setup attribute with the value which will change the negotiated setup role